inventory/host_vars/ofn.kandu.co.za/config.yml

```diff
@@ -1,6 +0,0 @@
----
-
-domain: ofn.kandu.co.za
```

Comment:
That server still exists. What's happening with it? Is there a handover, is Kandu still alive or do we have two instances now?

Reply:
They're basically migrating to the new domain, and ofn-install isn't really used on the existing server anyway.